.gitmodules

```
@@ -4,4 +4,4 @@
 
 [submodule "build"]
 path = build
-url = https://github.com/upbound/build
+	url = https://github.com/crossplane/build
```

```suggestion
url = https://github.com/crossplane/build
```

Indentation seems unintentional to me.